lxd/networks_utils.go

```diff
@@ -1017,17 +1017,55 @@ func networkUpdateStatic(s *state.State, networkName string) error {
 	return nil
 }
 
-func networkSysctl(path string, value string) error {
+func networkSysctlGet(path string) (int, error) {
```

Comment:
sysctls aren't always integers so this should really be returning a string

Reply:
fixed